libraries/key-identifier/src/key-identifier.js

```diff
@@ -59,6 +59,10 @@ export default class KeyIdentifier {
     return `${this.derivationPath} (${this.derivationAlgorithm})`
   }
 
+  [Symbol.toStringTag]() {
```

Comment:
```suggestion
  get [Symbol.toStringTag]() {
```

Reply:
yeah the get is required, missed it: https://github.com/ExodusMovement/exodus-oss/pull/116/commits/f0f7e4d1677ae65b068f8685418157edada6b08a